examples/spin-chain-model.ipynb

```diff
@@ -34,27 +34,43 @@
    "metadata": {},
```

Comment:
<p>is this Bloch sphere rendering?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>ok it is.</p>